### DISCUSSION TITLE
deep supervision loss

### DISCUSSION THREAD

**Discussion by cugwu** [Project-MONAI/tutorials#1117] [Q&A] [answered]
For the deep supervision implemented in dynamic unet of MONAI (https://docs.monai.io/en/stable/_modules/monai/networks/nets/dynunet.html). Is there a particular loss to apply or I just do the sum of the loss of each of the deep supervision heads?

**Comment by yiheng-wang-nv** [comment, marked as answer]
Hi @cugwu , the loss used is in: https://github.com/Project-MONAI/tutorials/blob/608795fd6578b84eef57b6c8f02dd7ce2a51a5c4/modules/dynunet_pipeline/trainer.py#L56
I think you can also try to adjust the weights for different resolutions, not sure if there are other methods.
